File: mesh/android/ExoPlayerMeshJava/app/src/main/java/io/streamroot/ctl/delivery/client/mesh/exoplayermesh/MainActivity.java
package io.streamroot.ctl.delivery.client.mesh.exoplayermesh;

import android.content.Intent;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import io.streamroot.ctl.delivery.client.mesh.exoplayermesh.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        final ActivityMainBinding binding = ActivityMainBinding.inflate(getLayoutInflater());
        binding.playButton.setOnClickListener(v -> startActivity(new Intent(this, PlayerActivity.class)));
        setContentView(binding.getRoot());
    }
}

File: mesh/android/ExoPlayerMesh/app/src/main/java/io/streamroot/ctl/delivery/client/mesh/exoplayermesh/MainActivity.kt
package io.streamroot.ctl.delivery.client.mesh.exoplayermesh

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import io.streamroot.ctl.delivery.client.mesh.exoplayermesh.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ActivityMainBinding.inflate(layoutInflater).apply {
            this.playButton.setOnClickListener {
                startActivity(Intent(this@MainActivity, PlayerActivity::class.java))
            }

            setContentView(this.root)
        }
    }
}